### DISCUSSION TITLE
How am I suppose to debug this?

### DISCUSSION THREAD

**Discussion by Marchiuzzz** [nuxt/nuxt#16251] [Questions]
I get a generic error while using ssr rendering (if I switch ssr flag to false in the config, everything works fine). Why do I see "missing stack frames"? Am I suppose to enable the extra dubug info somehow? As of now it's impossible to find what is wrong with my code...

![image](https://user-images.githubusercontent.com/73941456/204595208-73f2fb62-7e6e-46e0-90e7-ccbb73e13f5f.png)

**Comment by wobsoriano** [comment]
What modules are you trying to import?

**Reply by Marchiuzzz** [reply]
each vue file has an "export" keyword, that's the issue...I don't know what is causing this and the error window is useless

**Comment by memic84** [comment]
Have you tried setting `debug: true` in nuxt.config?

**Reply by Marchiuzzz** [reply]
this is already enabled
